11-change-ppp-mtu-debug.sh

```diff
@@ -30,17 +30,18 @@ while true; do
           echo $pinterface is one we should be checking
           # Check to see if we need to update the config file
           echo Checking MTU for $pinterface
-          pmtu=$(grep $(($PTARGET-8)) /etc/ppp/peers/$pinterface)
+          pmtu=$(grep 'mtu $(($PTARGET))' /etc/ppp/peers/$pinterface)
```

Comment:
look for mtu 1500 or whatever the PTARGET is instead of just PTARGET-8 (1492) because there is no guarantee that will actually be the value. also if someone has other changes with the same number somewhere else in this file that would cause problems. so i changed it to look for exactly the target mtu value. i assumed the mru would be the same which i think is fine. 

I also made it look for the correct rather than the wrong because ptarget-8 is never going to be true if you have a ptarget of like 1444